### ISSUE TITLE
Misspelled Word Not Caught in TXT File

### ISSUE THREAD

**Issue by teschste** [streetsidesoftware/vscode-spell-checker#3575]
This is admittedly a minor point but is nonetheless annoying, as I have really been enjoying this spellchecker.

In a text file associated with one of my projects, I noticed that "th" (an incomplete "the") is not flagged as misspelled.

I have checked both the custom words I have added to my personal settings, as well as the custom words unique to this project, to ensure I did not accidentally add this as a word, and confirmed I did not.

Why is "th" not being caught as a misspelling?  Can I force it to be caught?

Thanks!

**Comment by Jason3S** [comment]
@teschste,

There is a couple of things going on here
- The `cSpell.minWordLength` setting tells the spell checker to ignore not-found words shorter than 4 characters by default. 
- Since `th` appears in multiple dictionaries as a word (it is a keyword in Html and Markdown), it still would not have been reported.

You can force it to be caught using the settings by adding `th->the` to `cSpell.flagWords`.

<img width="1182" alt="image" src="https://github.com/user-attachments/assets/0e3878ab-0774-408b-b30d-7fdbcf96d512">

You can set `cSpell.minWordLength` to 2 to catch other possible issues, but it might be too noisy.

<img width="1191" alt="image" src="https://github.com/user-attachments/assets/eff93a2b-cb95-41d9-9f86-8c2cda287f1c">

Both of these settings can be added to a `cspell.config.yaml` or `cspell.json` file:

**`cspell.config.yaml`**

```yaml
flagWords:
  - th->the
minWordLength: 2
```

**`cspell.json`**

```json
{
  "flagWords": ["th:the"],
  "minWordLength": 2
}
```
